subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheEnablementIntegrationTest.groovy

```diff
@@ -42,6 +42,23 @@ class ConfigurationCacheEnablementIntegrationTest extends AbstractConfigurationC
         "system property" | ENABLE_SYS_PROP
     }
 
+    def "configuration cache is not supported yet for giving detailed information about a specific task"() {
```

Comment:
@scaventz Could you please move this test case to ConfigurationCacheIntegrationTest together with the other help test?

Also, since we are discarding the cache entry on the first run, and don't expect change in behavior for the second run, no need to try twice, right?

Reply:
@abstratt Thanks for your review. Ah... I'm not pretty sure I understand you correctly(pardon me if I don't), but I updated it, please take a look...

Reply:
@scaventz Thanks for contributing this fix. However, we may be taking a different approach for addressing this issue.

Instead of opting out of CC, we would like to make the Help task CC-friendly, mostly to promote the implementation of CC-friendly tasks by the community. While we look into that, we will put this PR on hold for a bit. We may come back and merge it, or it may no longer be needed.